packages/compiler-cli/ngcc/src/packages/entry_point_manifest.ts

```diff
@@ -128,6 +128,18 @@ export class EntryPointManifest {
   }
 }
 
+/**
+ * A specialized implementation of the `EntryPointManifest` that always returns `null` rather than a
+ * set of entry-points from a manifest file.
+ *
+ * This is used if you want to ignore a manifest file that is written to disc.
+ * Note that it will still write a new manifest file with the entry-points that were found by
+ * walking the directories.
+ */
+export class NullEntryPointManifest extends EntryPointManifest {
```

Comment:
May I propose `IgnoredEntryPointManifest`, as it's not entirely a null implementation.

```suggestion
export class IgnoredEntryPointManifest extends EntryPointManifest {
```

Reply:
Can I counter propose `InvalidatingEntryPointManifest`, since that is what it is doing?

Reply:
Sure, go ahead 👍